Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -208,6 +206,21 @@ void GameFile::SaveCache()
   // TODO
 }
 
+QString GameFile::GetFileName() const
+{
+  return QFileInfo(m_path).fileName();
```

Comment:
That was an issue before this, but yeah, they should be. I pushed another commit adding the `<QFileInfo>` include. (The code that uses `<QFile>` is unrelated to this change and will be removed in the caching PR.)